packages/mdc-ripple/_keyframes.scss

```diff
@@ -20,46 +20,33 @@
 
 @keyframes mdc-ripple-fg-radius-in {
   from {
-    transform: translate(0) scale(1);
```

Comment:
Why is this removed? Don't we need a fallback for browsers that don't support var?